docs/developing-component.md

```diff
@@ -22,6 +22,8 @@ git clone https://github.com/dapr/components-contrib.git dapr/components-contrib
 
 ```
 
+Note. When adding commits to the components-contrib repo, ensure you add `--signoff` or `-s` to the commits otherwise the PR you submit may be blocked.
```

Comment:
Please add something about the DCO requirement and the actions to take as seen here instead:
https://github.com/dapr/dapr/blob/master/CONTRIBUTING.md

Reply:
Done